falcon/testing/client.py

```diff
@@ -274,6 +274,11 @@ def encoding(self) -> Optional[str]:
         """
         return self._encoding
 
+    @property
+    def content_type(self) -> Optional[str]:
```

Comment:
maybe I should add a specific test for this that makes use also of the stream response